odoo/tests/test-uninstalls.py

```diff
@@ -56,6 +56,10 @@ def parse_args():
         help="Comma-separated list of paths to directories containing extra Odoo modules")
     parser.add_argument("--uninstall", "-U", type=str,
         help="Comma-separated list of modules to uninstall/reinstall")
+    parser.add_argument("--test-cow-views", action='store_true',
+        help="Test COW views deletion during module operations")
+    parser.add_argument("--test-all-l10n", action='store_true',
+        help="Test the installation of all l10n_* modules")
```

Comment:
That sucks.  In particular, the first test miserably crashes if the database does not have `test_website` installed...

What about keeping those test functions in their respective module, and provide some simple discovery mechanism to retrieve them, together with some tagging to enable/disable them?  For instance, we could provide a function decorator that collects those functions, and make this script (or another) import the modules containing those functions after loading the registry.